packages/client/src/components/Tabs/index.js

```diff
@@ -25,102 +33,162 @@ function TabPanel(props) {
         </div>
     );
 }
-
 TabPanel.propTypes = {
     children: PropTypes.node,
     index: PropTypes.any.isRequired,
     value: PropTypes.any.isRequired
 };
 
-function a11yProps(index) {
-    return {
-        id: `tab-${index}`,
-        'aria-controls': `tabpanel-${index}`
-    };
-}
-
-const ProcessTabs = withStyles({
+const AntTabs = withStyles({
+    root: {
+        borderBottom: '1px solid #e8e8e8'
+    },
     indicator: {
-        display: 'flex',
-        backgroundColor: '#F2BE32',
-        justifyContent: 'center',
-        '& > span': {
-            maxWidth: 40,
-            width: '100%'
-        }
+        backgroundColor: '#F2BE32'
     }
 })(Tabs);
-
-const useStyles = makeStyles({
+const AntTab = withStyles((theme) => ({
     root: {
-        flexGrow: 1,
-        backgroundColor: 'transparent',
-        display: 'flex',
-        height: 448
+        textTransform: 'none',
+        minWidth: 200,
+        fontWeight: theme.typography.fontWeightRegular,
+        // marginRight: theme.spacing(4),
+        fontFamily: [
+            '-apple-system',
+            'BlinkMacSystemFont',
+            '"Segoe UI"',
+            'Roboto',
+            '"Helvetica Neue"'
+        ].join(','),
+        '&:hover': {
+            color: '#003c5e',
+            opacity: 1
+        },
+        '&$selected': {
+            color: '#003c5e',
+            fontWeight: theme.typography.fontWeightBold
+        },
+        '&:focus': {
+            color: '#003c5e'
+        }
     },
-    tabs: {
-        borderRight: '1px solid rgba(0, 0, 0, 0.12)'
+    selected: {}
+}))((props) => <Tab disableRipple {...props} />);
+const useStyles2 = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1
     },
-    process: {
-        textTransform: 'none',
-        color: 'var(--text)',
-        backgroundColor: 'var(--background)',
-        borderRight: '3px solid #F2BE32',
-        borderLeft: '3px solid #F2BE32',
-        borderTop: '3px solid #F2BE32',
-        borderBottom: '3px solid #F2BE32'
+    padding: {
+        padding: theme.spacing(3)
     }
-});
+}));
 
-const selected = {
-    '--background': '#F2BE32',
-    '--text': '#fff'
-};
+// const ProcessTabs = withStyles({
```

Comment:
Let's remove commented code.

Reply:
It was the previously working Vertical Tabs. I will fix it later, but I need to retain most of the formatting because I don't want to redo it XD.